docker-compose.yaml

```diff
@@ -48,6 +48,8 @@ services:
       - POSTGRES_USER=${DATABASE_USER}
     volumes:
       - db:/var/lib/postgresql/data
+    ports:
+      - 5432:5432
```

Comment:
Just double checking - this is something you want to commit to master, right?

Reply:
No it is not, I am using that to be able to query my local DB when running docker-compose but I forget to remove it.